content/next/index.md

```diff
@@ -66,22 +68,25 @@ see the issue for details on the typos fixed
 
 fixes issue #12
 ```
+
 ## Specification
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-1. Commits MUST be prefixed with a type, which consists of a noun, `feat`, `fix`, etc., followed by a colon and a space.
+1. Commits MUST be prefixed with a type, which consists of a noun, `feat`, `fix`, etc., followed
+  by an OPTIONAL scope, and a REQUIRED terminal semicolon and space.
 1. The type `feat` MUST be used when a commit adds a new feature to your application or library.
 1. The type `fix` MUST be used when a commit represents a bug fix for your application.
-1. An optional scope MAY be provided after a type. A scope is a phrase describing a section of the codebase enclosed in parenthesis, e.g., `fix(parser):`
-1. A description MUST immediately follow the type/scope prefix.
-The description is a short description of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
+1. An optional scope MAY be provided after a type. A scope MUST consist of a noun describing a
```

Comment:
`An optional scope` => `A scope`

Since `optional` and `MAY` feel redundant.

Reply:
agreed 👍 good catch.